lib/rubocop/cop/style/hash_except.rb

```diff
@@ -42,162 +42,26 @@ module Style
       #   {foo: 1, bar: 2, baz: 3}.except(:bar)
       #
       class HashExcept < Base
-        include RangeHelp
+        include HashSliceExcept
         extend TargetRubyVersion
         extend AutoCorrector
 
         minimum_target_ruby_version 3.0
```

Comment:
`Hash#slice` was [added in Ruby 2.5](https://www.ruby-lang.org/en/news/2017/12/25/ruby-2-5-0-released/), I don't know if I should add `minimum_target_ruby_version 2.5` since currently required Ruby version is >= 2.7?

Reply:
> RuboCop can still inspect code all the way down to Ruby 2.0

That's impressive. Thanks, updated.